core/domain/html_validation_service.py

```diff
@@ -846,7 +847,7 @@ def add_dimensions_to_image_tags(exp_id, html_string):
                 'Exploration %s failed to load image: %s' %
                 (exp_id, image['filepath-with-value'].encode('utf-8')))
             raise e
-    return python_utils.convert_to_bytes(soup).replace('<br/>', '<br>')
+    return python_utils.STR(soup).replace('<br/>', '<br>')
```

Comment:
What is the difference between STR and convert_to_bytes?

Reply:
I can answer this one: `STR` is the standin for Python 3's `str` -- that is to say: `unicode`. So it's essentially use `unicode(soup)`.

`convert_to_bytes` is specifically for returning a `bytes` instance, since the single-argument constructor (`bytes(s)`) isn't robust enough to take an arbitrary value.

Reply:
Oh dear, that's actually kinda confusing. Would it be possible to rename STR to something like to_unicode() or convert_to_unicode()? I had interpreted it as exactly the opposite.

Reply:
Mm I don't think that's a good idea simply because `str` should and will continue to mean unicode in Python 3 and into the future. If anything, we should be forgetting about `unicode` as a concept and moving towards thinking of them as `str`ings.

Reply:
I would generally agree, but the problem is that, in the meantime, we are working with python 2, which has a different interpretation of "str". So I think we should preserve clarity as much as possible, and the use of the term unicode would at least be unambiguous (in either version).

Reply:
Should I name it UNICODE then? As to_unicode would mean I am converting it to unicode while it can be used in other cases too. For example, `isinstance(var, UNICODE)`